src/rules/no_window_prefix.rs

```diff
@@ -19,6 +21,8 @@ const CODE: &str = "no-window-prefix";
 const MESSAGE: &str = "For compatibility between the Window context and the Web Workers, calling Web APIs via `window` is disallowed";
 const HINT: &str =
   "Instead, call this API via `self`, `globalThis`, or no extra prefix";
+const FIX_GLOBAL_THIS_DESC: &str = "Rename window to globalThis";
+const FIX_SELF_DESC: &str = "Rename window to self";
```

Comment:
Does it make sense to have `self` here? Maybe this should only be `globalThis`? If we should keep this, then maybe the no_window rule should also have this?

Reply:
I think only `globalThis` should suffice.

Reply:
I agree that `globalThis` is enough here